### PR TITLE
Update generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent.md

### PR DESCRIPTION
**Pull request by poonam-lamba**
I ran the steps today on mac OS Catalina and it did not work for me. I had to make two changes that I updated in the doc.

### Check off the following:
- [ ] All of the tests are passing.
- [ ] I have reviewed my changes in staging.
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [ ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
Thanks for opening a pull request @poonam-lamba! I'll get this triaged for review ✨

File: content/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent.md [hunk 1]
```diff
@@ -88,14 +88,19 @@ Before adding a new SSH key to the ssh-agent to manage your keys, you should hav
         UseKeychain yes
         IdentityFile ~/.ssh/id_rsa
       ```
+  {% note %}
 
+  **Note:** If you are not using the passphrase and if you get an error then try removing the UseKeychain line.
+  
+  {% endnote %}
```
Comment (hubwriter) [inline content/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent.md:95]: To render correctly, the note box needs a blank line above the opening tag, and more indentation. See [the staging deployment](https://docs-908--patch-1.herokuapp.com/en/free-pro-team@latest/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent#adding-your-ssh-key-to-the-ssh-agent). I'd also suggest simplifying the advice about `UseKeychain`:
 
```suggestion

     {% note %}

     **Note:** If you chose not to add a passphrase to your key, you should omit the `UseKeychain` line.
  
     {% endnote %}
```

File: content/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent.md [hunk 2]
```diff
@@ -88,14 +88,19 @@ Before adding a new SSH key to the ssh-agent to manage your keys, you should hav
...
+  
 3. Add your SSH private key to the ssh-agent and store your passphrase in the keychain. {% data reusables.ssh.add-ssh-key-to-ssh-agent %}
    ```shell
    $ ssh-add -K ~/.ssh/id_rsa
   ```
   {% note %}
 
-  **Note:** The `-K` option is Apple's standard version of `ssh-add`, which stores the passphrase in your keychain for you when you add an ssh key to the ssh-agent.
+  **Note:** The `-K` option is Apple's standard version of `ssh-add`, which stores the passphrase in your keychain for you when you add an ssh key to the ssh-agent.If you leave your passphrase blank, run the command without -K option.
```
Comment (hubwriter) [inline content/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent.md:103]:
```suggestion
  **Note:** The `-K` option is Apple's standard version of `ssh-add`, which stores the passphrase in your keychain for you when you add an ssh key to the ssh-agent. If you chose not to add a passphrase to your key, run the command without the `-K` option. 
```

**Review by hubwriter — CHANGES_REQUESTED** [review]
@poonam-lamba - Many thanks for creating this pull request. Although we direct people to use a passphrase I think it's helpful to provide this extra information for people who use a key without a passphrase.

I've suggested a few small changes.

Thanks again for helping to improve GitHub documentation. 👍

**Review by hubwriter — APPROVED** [review]

**Comment by hubwriter** [comment]
No response from author to my commit here
https://github.com/github/docs/pull/908/commits/7566ecb23ca95cdf827896831a9ce7196e57d23e
so I'm going to assume all is good here and merge this now.

**Review by hubwriter — APPROVED** [review]

**Pull request merged**